src/estimator.cpp

```diff
@@ -181,30 +188,49 @@ void Estimator::run()
     // Below Eq. 45 Mahony Paper
     turbomath::Quaternion q_tilde = q_acc_inv * state_.attitude;
     // Correction Term of Eq. 47a and 47b Mahony Paper
-    // w_acc = 2*s_tilde*v_tilde
-    w_acc.x = -2.0f*q_tilde.w*q_tilde.x;
-    w_acc.y = -2.0f*q_tilde.w*q_tilde.y;
-    w_acc.z = 0.0f; // Don't correct z, because it's unobservable from the accelerometer
-
-    // integrate biases from accelerometer feedback
-    // (eq 47b Mahony Paper, using correction term w_acc found above
-    bias_.x -= ki*w_acc.x*dt;
-    bias_.y -= ki*w_acc.y*dt;
-    bias_.z = 0.0;  // Don't integrate z bias, because it's unobservable
-  }
-  else
-  {
-    w_acc.x = 0.0f;
-    w_acc.y = 0.0f;
-    w_acc.z = 0.0f;
+    // w_err = 2*s_tilde*v_tilde
+    w_err.x = -2.0f*q_tilde.w*q_tilde.x;
+    w_err.y = -2.0f*q_tilde.w*q_tilde.y;
+    w_err.z = 0.0f; // Don't correct z, because it's unobservable from the accelerometer
   }
 
   if (attitude_correction_next_run_)
   {
+    turbomath::Vector xhat_BW, yhat_BW, zhat_BW;
```

Comment:
Thanks for working all this out.

What do you think about including Eigen as a dependency?  Perhaps that's out of the scope of this PR, but I think it would be nice to replace `turbomath::Quat` and `turbomath::Vector3` with Eigen.

Reply:
Also, is there a chance you could write the associated latex code here as a comment?  It's a little too big to grok at once :)

Reply:
I agree that using Eigen would be nice.